resources/lang/id/messages.json

```diff
@@ -0,0 +1,77 @@
+{
+  "greeting": "Halo,",
+  "totalCards": {
+    "incomes": "Pemasukan",
+    "outcomes": "Pengeluaran",
+    "total": "Saldo"
+  },
+  "nav": {
+    "list": "Transaksi",
+    "register": "Tambah",
+    "resume": "Ringkasan"
+  },
+  "lastTransactions": "Transaksi terakhir",
+  "deleteModal": {
+    "heading": "Kamu yakin ingin menghapus &#34;{ transactionName }&#34;?",
+    "warning": "Tindakan ini tidak bisa dibatalkan.",
```

Comment:
Hmm, it seems more appropriate to "dikembalikan" instead of "dibatalkan", but this is not a big problem, just a language inaccuracy :D